lenses/passwd.aug

```diff
@@ -28,7 +28,7 @@ let comment    = Util.comment
 let empty      = Util.empty
 let dels       = Util.del_str
 
-let word       = Rx.word
+let word       = /[_.A-Za-z0-9][-\@_.A-Za-z0-9]*\$?/
```

Comment:
Why is `$` allowed at the end of this regexp ? From further down, it seems to be pulling double-duty as a regexp for passwords, and for user names.